allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -85,7 +85,17 @@ def list_api_methods(self) -> List[Tuple[str, str]]:
 
     # ========================= 'get' methods ==========================
 
-    def get_reward_rate(self):
+    def get_reward_rate(self) -> np.ndarray:
+        """ Get the reward rate of the subject for the task calculated over a
```

Comment:
@dougollerenshaw Just wanted to double check that this docstring looks fine to you as well.

Reply:
There is a typo just below this point (on line 96): reard should be reward.
I would also suggest listing the units (rewards/minute) on line 97.

Reply:
Fixed and updated.